$OpenBSD$

Now b.root-servers.net is 192.228.79.201.
Now j.root-servers.net is 192.158.128.30. (SERIAL 2002110501)
Use m.root-servers.net first for Japenese user.
Move a.root-servers.net to last.

--- dnsroots.global.orig	Mon Feb 12 06:11:45 2001
+++ dnsroots.global	Fri Jan 30 01:21:29 2004
@@ -1,5 +1,5 @@
-198.41.0.4
-128.9.0.107
+202.12.27.33
+192.228.79.201
 192.33.4.12
 128.8.10.90
 192.203.230.10
@@ -7,7 +7,6 @@
 192.112.36.4
 128.63.2.53
 192.36.148.17
-198.41.0.10
+192.58.128.30
 193.0.14.129
 198.32.64.12
-202.12.27.33
